src/java.base/share/classes/java/security/doc-files/security-related-system-properties.html

```diff
@@ -0,0 +1,197 @@
+<H1 style="text-align:center">Security-Related System Properties</H1>
+<p>
+    There are several system properties used to
+    alter the mechanisms and behavior of the various classes of the
+    <code>java.security</code> package. Some are checked only once at startup of the VM,
+    and therefore are best set using the <code>-D</code> option of the java command,
+    while others have a more dynamic nature and can also be changed using
+    the <a href="../../lang/System.html#setProperty(java.lang.String,java.lang.String)">System.setProperty()</a> API.
+    </P>
+<a id="security.debug"></a>
+<H2>Debug</H2>
+<p><B>{@systemProperty java.security.debug}</B><BR></p>
+        <p> To monitor security access, you can set the <code>java.security.debug</code> system property,
+            which determines what trace messages are printed during execution. The value of the property is one or more options separated by a comma.
+            Some options also have additional sub-options; see the list below for more details on the syntax of each.
+            The following table lists <code>java.security.debug</code> options table:</p>
+
+<table id="debug">
+    <caption><b>Security Debug Options</b></caption>
+    <thead>
+    <tr>
+        <th scope="col">Option</th>
+        <th scope="col">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <th scope="row"><code>all</code></th>
+        <td><code>Turn on all the debugging options</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>certpath</code></th>
+        <td>Turns on debugging for the PKIX CertPathValidator and CertPathBuilder implementations. Use the ocsp option with the certpath option for OCSP protocol tracing. A hexadecimal dump of the OCSP request and response bytes is displayed.
+            You can use the following options with the certpath option:
+            <ul>
+                <li><code><b>ocsp:</b> Dump OCSP protocol exchanges</code></li>
+                <li><code><b>verbose:</b> Print additional debugging information</code></li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <th scope="row"><code>configfile</code></th>
+        <td><code>JAAS (Java Authentication and Authorization Service) configuration file loading</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>configparser</code></th>
+        <td><code>JAAS configuration file parsing</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>gssloginconfig</code></th>
+        <td><code>Java GSS (Generic Security Services) login configuration file debugging</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>jar</code></th>
+        <td><code>JAR file verification</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>jca</code></th>
+        <td><code>JCA engine class debugging</code></td>
+    </tr>
+    <tr>
+        <th scope="row"><code>keystore</code></th>
+        <td><code>Keystore debugging</code></td>
```

Comment:
Consider change to `KeyStore` to be consistent with rows below. I prefer them in `<code>`.